packages/babel-cli/bin/babel-node.js

```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env node
 
-require("../lib/babel-node");
+throw new Error("babel-node has been removed.");
```

Comment:
I was thinking we could just include it by requiring the new package instead? If we don't do this, we should at least say in the error message to install it.

Reply:
Can I modify the message to say "`babel-node` has been removed from `babel-cli`, please install it separately." If it's not appropriate, please suggest the proper error message.

Reply:
@hzoo Please let me know what should be done here?
 - [ ] Should I require `babel-node` package?
 - [ ] Should I just modify the error message?

Reply:
@hzoo Still waiting for your input 🙂

Reply:
for me, I would require the new dependency so it works the same

Reply:
Ok, so I added below to `packages/babel-cli/bin/babel-node.js` file
```js
#!/usr/bin/env node

require("babel-node");
```
and added the dependency in `package.json` file as `"babel-node": "7.0.0-beta.0"`. However when I try to execute the script using `babel-node`, I get following error:
```
Error: Cannot find module 'babel-node'
```
I am not sure what's going wrong 😔 Help please!